src/requirements.h

```diff
@@ -190,6 +210,8 @@ struct requirement_data {
         const requirement_data disassembly_requirements() const;
 
     private:
+        std::string id_;
```

Comment:
It was initially but I refactored to plain string when I realised why would an instance need to refer to itself?